admin/template_select.php

```diff
@@ -104,6 +105,16 @@
                 $template_split = new splitPageResults($_GET['page'], MAX_DISPLAY_SEARCH_RESULTS, $template_query_raw, $template_query_numrows);
                 $templates = $db->Execute($template_query_raw);
                 foreach ($templates as $template) {
+                  if (!isset($template_info[$template['template_dir']])) {
+                     $template_info[$template['template_dir']] = [
+                       'name' => $template['template_dir'],
+                       'version' => '', 
+                       'author' => '', 
+                       'description' => '',
+                       'screenshot' => '', 
+                       'missing' => true, 

```

Comment:
Not a fan of adding this `missing` element. Why not just call `continue` here, instead of later? Or unset the `$template` entry from the `$templates` array?

Reply:
If you do that then the line on the page showing the current template will be blank.